Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewController.swift

```diff
@@ -36,9 +33,9 @@ public final class GDSInformationViewController: BaseViewController, TitledViewC
             let heightConstraint: CGFloat
             
             if let value = viewModel.imageHeightConstraint {
-                heightConstraint = value + imagePaddingCompensation
+                heightConstraint = value + 11
             } else {
-                heightConstraint = defaultImageHeight
+                heightConstraint = 55
```

Comment:
what are these magic numbers? 👀

if we're hard-coding heights, please can we add a comment to explain why? it doesn't really follow our accessibility guidance

Reply:
These were hard coded to properties of the class, I've added a comment to explain